src/core/expressions.jl

```diff
@@ -1,4 +1,12 @@
+"""
+Expression for HydroPumpedStorage that keep track
```

Comment:
Please turn this and all following references to HydroPumpedStorage into hyperlinks to the docstring using `@extref`: http://juliadocs.org/DocumenterInterLinks.jl/stable/